docs/predict-api/v2/required_api.md

```diff
@@ -281,14 +281,12 @@ code. The inference response object, identified as
 
     $inference_response =
     {
-      "model_name" : $string,
```

Comment:
So the spec states to return model_version but not model_name? It may be a nice convenience to have the model_name. WDYT about updating the spec?

Reply:
I think it would be good to have the model name, and that is what I originally proposed. But at some point along the way there was the suggestion that because it wasn't absolutely necessary it should be removed. This just fixes the doc issue that I didn't remove it consistently for both HTTP and GRPC. I'm happy to add it back in both places.

Reply:
Let's keep this in the spec please. Not absolutely necessary, but does mitigate potential pain.

Reply:
Updated.